storey/targets.py

```diff
@@ -533,6 +533,8 @@ class ParquetTarget(_Batching, _Writer):
     :param storage_options: Extra options that make sense for a particular storage connection, e.g. host, port,
         username, password, etc., if using a URL that will be parsed by fsspec, e.g., starting
         "s3://”, "gcs://”. Optional.
+    :param single_file: If True, all the partitioned data will be written to a single file named {name}.parquet in
```

Comment:
So `{name}` is the name of the target? i.e. `ParquetTarget.parquet` or `my_parquet_target.parquet`? Not sure if that's the best way to go, but anyway this needs to be clarified if we keep this behaviour.

Reply:
I will go with `target.parquet `